### PR TITLE
Added CommandGoToOtherEndOfHiglightedText - #526

### LINKED ISSUE #526: Visual mode: "o" - move to other end of marked area
### What did you do?

Mark text in Visual Mode and press `o`.

### What did you expect to happen?

Wanted to move cursor to other end of marked area.

### What happened instead?

Nothing.

### Technical details:

* VSCode Version: Version 1.4.0-insider
  * Commit: 825a1e71a436007d88796cb1cf0bad85ad534e53
  * Date: 2016-07-27T07:11:53.180Z
* VsCodeVim Version: *0.1.3*
* OS: Win7

### PR DESCRIPTION
**Pull request by Platzer**
Sry, i tried to fix #526 but i'm not a typescript nor js nor web developer and not really familiar with github pull request and your contribution guidelines. Just take a look, its a simple fix but i'm not sure if i created any side-effects.

### CODE REVIEW

File: src/actions/actions.ts [hunk 1]
```diff
@@ -1194,6 +1194,22 @@ class CommandUnfoldAll extends BaseCommand {
 }
 
 @RegisterAction
+class CommandGoToOtherEndOfHighlightedText extends BaseCommand {
+  modes = [ModeName.Visual];
```
Comment (rebornix) [inline src/actions/actions.ts]: Include `ModeName.VisualLine` as well?

File: src/actions/actions.ts [hunk 2]
```diff
@@ -1194,6 +1194,22 @@ class CommandUnfoldAll extends BaseCommand {
...
+  keys = ["o"];
+
+  public async exec(position: Position, vimState: VimState): Promise<VimState> {
+    let t1 = vimState.cursorPosition;
```
Comment (rebornix) [inline src/actions/actions.ts]: You would like below syntax sugar

```
[a, b] = [b, a]
```

**Comment by rebornix** [comment]
@Platzer you are really doing good and thanks!

**Comment by Platzer** [comment]
Applied all changes, thx for your feedback!

**Comment by johnfn** [comment]
A two line function body. That's what I like to see. Beautiful!

**Pull request merged**